redbot/core/core_commands.py

```diff
@@ -2059,7 +2060,7 @@ async def custominfo(self, ctx: commands.Context, *, text: str = None):
         else:
             await ctx.send(_("Text must be fewer than 1024 characters long."))
 
-    @_set.command()
+    @_set.group(invoke_without_command=True)
     @checks.is_owner()
     async def api(self, ctx: commands.Context, service: str, *, tokens: TokenConverter):
         """Set various external API tokens.
```

Comment:
There is no mention subcomments of `[p]help set`, I would reccomend this command is broken up into 3 subcommands.

Reply:
I think it would be better to just mention in `[p]set api`'s docstring that it's also for listing and deleting (in whatever way that would be worded) if there's an issue with clarity, so that all the commands related to API tokens are still grouped together.

Reply:
I've updated the `[p]`set api` docstring to reflect the new subcommands.